### DISCUSSION TITLE
How to Integrate With Eventmesh

### DISCUSSION THREAD

**Discussion by Roc-00** [open-telemetry/community#769] [Community Discussions]
EventMesh is a dynamic cloud-native eventing infrastruture used to decouple the application and backend middleware layer, which supports a wide range of use cases that encompass complex multi-cloud, widely distributed topologies using diverse technology stacks.How can we Integrate with eventmesh?

**Comment by SergeyKanzhelev** [comment]
Are you looking for specific libraries support or understand how to model event mesh using OpenTelemetry terminology and types?

**Reply by Roc-00** [reply]
Are there any specific libraries support?

**Reply by SergeyKanzhelev** [reply]
this is what I'm asking - do you have any libraries in mind? Or this is a general question?
